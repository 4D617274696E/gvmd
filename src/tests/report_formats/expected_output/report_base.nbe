timestamps|||scan_start|Thu Aug  5 07:52:45 2010|
results|1.1.1.1|1.1.1.1|microsoft-ds (445/tcp)|1.3.6.1.4.1.25623.1.0.900233|Security Hole|\n\n  Overview: This host has critical security update missing according to\n  Microsoft Bulletin MS09-001.\n\n  Vulnerability Insight:\n  The issue is due to the way Server Message Block (SMB) Protocol software\n  handles specially crafted SMB packets.\n\n  Impact: Successful exploitation could allow remote unauthenticated attackers\n  to cause denying the service by sending a specially crafted network message\n  to a system running the server service.\n\n  Impact Level: System/Network\n\n  Affected Software/OS:\n  Microsoft Windows 2K Service Pack 4 and prior.\n  Microsoft Windows XP Service Pack 3 and prior.\n  Microsoft Windows 2003 Service Pack 2 and prior.\n\n  Fix: Run Windows Update and update the listed hotfixes or download and\n  update mentioned hotfixes in the advisory from the below link,\n  http://www.microsoft.com/technet/security/bulletin/ms09-001.mspx\n\n  References:\n  http://www.milw0rm.com/exploits/6463\n  http://www.microsoft.com/technet/security/bulletin/ms09-001.mspx\n\n  CVSS Score:\n    CVSS Base Score     : 7.1 (AV:N/AC:M/Au:NR/C:N/I:N/A:C)\n    CVSS Temporal Score : 5.6\n  Risk factor : High\nCVE : CVE-2008-4114, CVE-2008-4834, CVE-2008-4835\nBID : 31179\n
results|1.1.1.1|1.1.1.1|netbios-ns (137/udp)|1.3.6.1.4.1.25623.1.0.10150|Security Warning|The following 4 NetBIOS names have been gathered :\n POLYNOE         = This is the computer name registered for workstation services by a WINS client.\n INTEVATION      = Workgroup / Domain name\n POLYNOE         = Computer name\n INTEVATION      = Workgroup / Domain name (part of the Browser elections)\nThe remote host has the following MAC address on its adapter :\n   00:11:2f:0e:75:11\n\nIf you do not want to allow everyone to find the NetBios name\nof your computer, you should filter incoming traffic to this port.\n\nRisk factor : Medium\nCVE : CAN-1999-0621\n
results|1.1.1.1|1.1.1.1|vnc-http (5800/tcp)|1.3.6.1.4.1.25623.1.0.10758|Security Warning|\nThe remote server is running VNC.\nVNC permits a console to be displayed remotely.\n\nSolution: Disable VNC access from the network by \nusing a firewall, or stop VNC service if not needed.\n\nRisk factor : Medium\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.102002|Security Note|ICMP based OS fingerprint results: \n\nMicrosoft Windows 2003 Server Enterprise Edition (accuracy 100%)\nMicrosoft Windows 2003 Server Standard Edition (accuracy 100%)\nMicrosoft Windows XP SP2 (accuracy 100%)\n\n\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.50282|Security Note|System identifier unknown: "bash: uname: command not found\n"\nTherefore no local security checks applied (missing list of installed packages) though ssh login provided and works\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.900239|Security Note|Open TCP ports are 445, 5900, 22, 135, 5800, 139\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.90003|Security Note|Slad_Fetch has no Result polled.\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.90002|Security Note|ChkRootKit: The Plugin was not present on target system, but selected to execute.\nClamAV: The Plugin was not present on target system, but selected to execute.\nHardware MB sensors: The Plugin was not present on target system, but selected to execute.\nHardware MB sensors: The Plugin was not present on target system, but selected to execute.\nLSOF: The Plugin was not present on target system, but selected to execute.\nMBSA: The Plugin was not present on target system, but selected to execute.\nNetstat: The Plugin was not present on target system, but selected to execute.\nOvaldi: The Plugin was not present on target system, but selected to execute.\nTiger: The Plugin was not present on target system, but selected to execute.\nTripwire HIDS: The Plugin was not present on target system, but selected to execute.\nSFC: The Plugin was not present on target system, but selected to execute.\nSnmptrapd: The Plugin was not present on target system, but selected to execute.\nSnort: The Plugin was not present on target system, but selected to execute.\nssh vulnkey: The Plugin was not present on target system, but selected to execute.\nSyslogWatch: The Plugin was not present on target system, but selected to execute.\n\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.80091|Security Note|\nSynopsis :\n\nThe remote service implements TCP timestamps.\n\nDescription :\n\nThe remote host implements TCP timestamps, as defined by RFC1323.\nA side effect of this feature is that the uptime of the remote \nhost can sometimes be computed.\n\nSee also :\n\nhttp://www.ietf.org/rfc/rfc1323.txt\n\nRisk factor : \n\nNone\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.51662|Security Note|Here is the route from 192.168.11.233 to 1.1.1.1\n\n192.168.11.233\n1.1.1.1\n\n
results|1.1.1.1|1.1.1.1|microsoft-ds (445/tcp)|1.3.6.1.4.1.25623.1.0.11011|Security Note|A CIFS server is running on this port\n
results|1.1.1.1|1.1.1.1|microsoft-ds (445/tcp)|1.3.6.1.4.1.25623.1.0.10394|Security Note|It was not possible to log into the remote host using user defined\nlogin/password combinations :\n\n
results|1.1.1.1|1.1.1.1|microsoft-ds (445/tcp)|1.3.6.1.4.1.25623.1.0.102011|Security Note|\nOverview:\nIt is possible to extract OS, domain and SMB server information\nfrom the Session Setup AndX Response packet which is generated\nduring NTLM authentication.\nDetected SMB workgroup: INTEVATION\nDetected SMB server: Windows 2000 LAN Manager\nDetected OS: Windows 5.1\n\n
results|1.1.1.1|1.1.1.1|netbios-ssn (139/tcp)|1.3.6.1.4.1.25623.1.0.11011|Security Note|An SMB server is running on this port\n
results|1.1.1.1|1.1.1.1|ntp (123/udp)|1.3.6.1.4.1.25623.1.0.10884|Security Note|\n  A NTP (Network Time Protocol) server is listening on this port.\n\n  Risk factor : Low\n
results|1.1.1.1|1.1.1.1|ssh (22/tcp)|1.3.6.1.4.1.25623.1.0.10330|Security Note|An ssh server is running on this port\n
results|1.1.1.1|1.1.1.1|ssh (22/tcp)|1.3.6.1.4.1.25623.1.0.90022|Security Note|It was possible to login using the SSH credentials supplied.\nHence local security check are enabled.\n
results|1.1.1.1|1.1.1.1|ssh (22/tcp)|1.3.6.1.4.1.25623.1.0.10267|Security Note|Remote SSH version : SSH-2.0-OpenSSH_5.4\n\nRemote SSH supported authentication : publickey,password,keyboard-interactive\n\n\n
results|1.1.1.1|1.1.1.1|ssh (22/tcp)|1.3.6.1.4.1.25623.1.0.100259|Security Note|\n\nOverview:\nThe remote SSH Server supports the following SSH Protocol Versions:\n\n1.99\n2.0\n\nSSHv2 Fingerprint: 59:10:f9:5c:e6:dd:3f:21:2b:b2:c8:68:03:f5:1e:55\n\n\nRisk factor : None\n\n
results|1.1.1.1|1.1.1.1|vnc (5900/tcp)|1.3.6.1.4.1.25623.1.0.10342|Security Note|\nSynopsis :\n\nThe remote host is running a remote display software (VNC)\n\nDescription :\n\nThe remote server is running VNC, a software which ermits a \nconsole to be displayed remotely.\n\nThis allows authenticated users of the remote host to take its \ncontrol remotely.\n\nSolution : \n\nMake sure the use of this software is done in accordance with your\ncorporate security policy, filter incoming traffic to this port.\n\nRisk factor : \n\nNone\n\nPlugin output :\nThe version of the VNC protocol is : RFB 003.003\n\n
results|1.1.1.1|1.1.1.1|vnc (5900/tcp)|1.3.6.1.4.1.25623.1.0.19288|Security Note|The remote VNC server chose security type #2 (VNC authentication)\n
results|1.1.1.1|1.1.1.1|vnc-http (5800/tcp)|1.3.6.1.4.1.25623.1.0.10330|Security Note|A web server is running on this port\n
results|1.1.1.1|1.1.1.1|vnc-http (5800/tcp)|1.3.6.1.4.1.25623.1.0.80109|Security Note|Here is the w3af report:\n[ Thu Aug  5 07:55:11 2010 - information ] Auto-enabling plugin: grep.error500\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ] plugins\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     audit sqli, xss\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     audit config sqli\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     audit config xss\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set checkStored True\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set numberOfChecks 3\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ] plugins\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     grep error500\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ] plugins\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     output textFile, console, gtkOutput\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     output config textFile\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set verbose False\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set fileName /tmp/openvas-w3af-1.1.1.1-5800.rep\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set httpFileName /tmp/openvas-w3af-1.1.1.1-5800.http\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set showCaller False\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     output config console\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set verbose False\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     output config gtkOutput\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ] plugins\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     discovery yahooSiteExplorer\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     discovery config yahooSiteExplorer\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         set resultLimit 300\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]         back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     back\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ] target\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     set target http://1.1.1.1:5800\n[ Thu Aug  5 07:55:11 2010 - Enabled plugins ]     back\n[ Thu Aug  5 07:55:11 2010 - error ] There is no point in searching yahoo site explorer for site: "1.1.1.1" . Yahoo doesnt index private pages.\n[ Thu Aug  5 07:55:11 2010 - information ] Found 1 URLs and 1 different points of injection.\n[ Thu Aug  5 07:55:11 2010 - information ] The list of URLs is:\n[ Thu Aug  5 07:55:11 2010 - information ] - http://1.1.1.1:5800\n[ Thu Aug  5 07:55:11 2010 - information ] The list of fuzzable requests is:\n[ Thu Aug  5 07:55:11 2010 - information ] - http://1.1.1.1:5800 | Method: GET\n[ Thu Aug  5 07:55:11 2010 - information ] Finished scanning process.\n[ Thu Aug  5 07:55:12 2010 - console ] termios error: (25, 'Inappropriate ioctl for device')\n[ Thu Aug  5 07:55:12 2010 - console ] \n[ Thu Aug  5 07:55:12 2010 - console ] termios error: (25, 'Inappropriate ioctl for device')\n[ Thu Aug  5 07:55:12 2010 - console ] \nBye.\n\n\n
results|1.1.1.1|1.1.1.1|epmap (135/tcp)|0|Log Message|Open port.
results|1.1.1.1|1.1.1.1|general/CPE|1.3.6.1.4.1.25623.1.0.810002|Log Message|No CPE identities could be determined.\n
results|1.1.1.1|1.1.1.1|general/tcp|1.3.6.1.4.1.25623.1.0.19506|Log Message|Information about this scan : \n\nOpenVAS version : 3.1.0.rc1.\nPlugin feed version : 201008041240\nType of plugin feed : Greenbone Security Feed\nScanner IP : 192.168.11.233\nPort scanner(s) : nmap \nPort range : default\nThorough tests : no\nExperimental tests : no\nParanoia level : 1\nReport Verbosity : 1\nSafe checks : yes\nMax hosts : 20\nMax checks : 4\nScan Start Date : 2010/8/5 7:52\nScan duration : 253 sec\n\n
results|1.1.1.1|1.1.1.1|microsoft-ds (445/tcp)|0|Log Message|Open port.
results|1.1.1.1|1.1.1.1|netbios-ssn (139/tcp)|0|Log Message|Open port.
results|1.1.1.1|1.1.1.1|ssh (22/tcp)|0|Log Message|Open port.
results|1.1.1.1|1.1.1.1|vnc (5900/tcp)|0|Log Message|Open port.
results|1.1.1.1|1.1.1.1|vnc-http (5800/tcp)|0|Log Message|Open port.
timestamps||1.1.1.1|host_start|Thu Aug  5 07:52:46 2010|
timestamps||1.1.1.1|host_end|Thu Aug  5 07:56:59 2010|
timestamps|||scan_end|Thu Aug  5 07:56:59 2010|
