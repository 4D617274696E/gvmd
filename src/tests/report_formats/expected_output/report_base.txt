I Summary
=========

This document reports on the results of an automatic security scan.
The report first summarises the results found.
Then, for each host, the report describes every issue found.
Please consider the advice given in each description, in order to rectify
the issue.

Scan started: Thu Aug  5 07:52:45 2010
Scan ended:   Thu Aug  5 07:56:59 2010

Host Summary
************

Host            High  Medium  Low  Log  False Positive
1.1.1.1            1       2   20    8               0
Total: 1           1       2   20    8               0


II Results per Host
===================

Host 1.1.1.1
************

Scanning of this host started at: Thu Aug  5 07:52:46 2010
Number of results: 31

Port Summary for Host 1.1.1.1
-----------------------------

Service (Port)          Threat Level
microsoft-ds (445/tcp)  High
netbios-ns (137/udp)    Medium
vnc-http (5800/tcp)     Medium
general/tcp             Low
netbios-ssn (139/tcp)   Low
ntp (123/udp)           Low
ssh (22/tcp)            Low
vnc (5900/tcp)          Low
epmap (135/tcp)         Log
general/CPE             Log


Issue:
------
NVT: Vulnerabilities in SMB Could Allow Remote Code Execution (958687) - Remote 
OID: 1.3.6.1.4.1.25623.1.0.900233
Threat: High (CVSS: 7.1)
Port: microsoft-ds (445/tcp)
Description: 


  Overview: This host has critical security update missing according to
  Microsoft Bulletin MS09-001.

  Vulnerability Insight:
  The issue is due to the way Server Message Block (SMB) Protocol software
  handles specially crafted SMB packets.

  Impact: Successful exploitation could allow remote unauthenticated attackers
  to cause denying the service by sending a specially crafted network message
  to a system running the server service.

  Impact Level: System/Network

  Affected Software/OS:
  Microsoft Windows 2K Service Pack 4 and prior.
  Microsoft Windows XP Service Pack 3 and prior.
  Microsoft Windows 2003 Service Pack 2 and prior.

  Fix: Run Windows Update and update the listed hotfixes or download and
  update mentioned hotfixes in the advisory from the below link,
  http://www.microsoft.com/technet/security/bulletin/ms09-001.mspx

  References:
  http://www.milw0rm.com/exploits/6463
  http://www.microsoft.com/technet/security/bulletin/ms09-001.mspx

  CVSS Score:
    CVSS Base Score     : 7.1 (AV:N/AC:M/Au:NR/C:N/I:N/A:C)
    CVSS Temporal Score : 5.6
  Risk factor : High
CVE : CVE-2008-4114, CVE-2008-4834, CVE-2008-4835
BID : 31179
